### PR TITLE
directory: split files and fix comments

### PR DESCRIPTION
**Pull request by runcom**
Signed-off-by: Antonio Murdaca <runcom@redhat.com>

### CODE REVIEW

File: directory/directory.go [hunk 1]
```diff
@@ -2,124 +2,22 @@ package directory
...
-	"github.com/containers/image/types"
 )
 
 // manifestPath returns a path for the manifest within a directory using our conventions.
 func manifestPath(dir string) string {
 	return filepath.Join(dir, "manifest.json")
 }
 
-// manifestPath returns a path for a layer tarball within a directory using our conventions.
+// lyaerPath returns a path for a layer tarball within a directory using our conventions.
```
Comment (sgotti) [inline directory/directory.go]: s/lyaerPath/layerPath/

Comment (runcom) [inline directory/directory.go]: :+1:

**Comment by runcom** [comment]
lgtm

**Comment by mtrmac** [comment]
:+1:

**Pull request merged**